repository-data/application/src/main/resources/hcm-config/namespaces/publicationsystem/resource.yaml

```diff
@@ -0,0 +1,81 @@
+---
+definitions:
```

Comment:
Worth a comment saying where this was copied from, why and what was changed?

Reply:
Comments are removed from YAML currently :(